### PR TITLE
Desktop version of my portfolio

### PR DESCRIPTION
**Pull request by Moise-code**
Desktop version of my portfolio :

In this project I have added:
responsiveness using media querries for two types of device frame; Mobile devices and desktop.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -52,11 +65,15 @@ <h3>I'am a Software Developer</h3>
   <section class="work">
     <div class="work-title">
       <h4>My Recent Works</h4>
+      <hr>
```
Comment (nedjwab) [inline index.html:68]: [ ] kindly change the `font color `of the technologies list and the title in this section to match the Figma template, the one used in the template is `#091E42` .🤓
Your design : 
headerta3ou

icons

Figma design : 
designicons
header

File: Styles/styles.css [hunk 2]
```diff
@@ -534,8 +552,690 @@ footer form textarea {
...
+    width: 794px;
+    vertical-align: middle;
+  }
+
+  .grid {
+    display: grid;
+  }
+
+  .collection {
+    display: grid;
```
Comment (nedjwab) [inline Styles/styles.css:807]: nice use of the grid and flexbox  🔥🔥

File: index.html [hunk 3]
```diff
@@ -52,11 +65,15 @@ <h3>I'am a Software Developer</h3>
...
     </div>
     <section class="grid">
       <div class="collection">
         <div class="work1">
+          <div>
           <img src="items/img-for-first-project.png" alt="">
+        </div>
+        <div class="work1-second-part">
```
Comment (nedjwab) [inline index.html:76]: [ ] kindly add a hover effect to your cards, so when we hover on with the mouse the description and dark opacity disappear and a See Project button appears at the bottom of the image and takes the whole image width mentioned button should take you to the Live Demo of the project or to the GitHub repository.👀
Your cards : 
hoverta3ou

Figma design : 
hoverdesign

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Moise-code 

Nice job✔, I liked your implementation 🔥, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.
- Good design 🔥🔥🔥 , keep it up.😉

## (Changes Required) Aspects to improve: ♻️
You did great implementing  the project requirements 🔥🔥, but you still have some issues:  
Check the comments below 👇
  
## [Optional] suggestions:

-  Nothing to mention. 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 4]
```diff
@@ -68,9 +85,10 @@ <h5>Multi-Post Stories</h5>
             <li>Bootstrap</li>
```
Comment (bobb-Rob) [inline index.html:85]: - [ ] [REQUIRED] As mentioned by the previous reviewer, please the border for the list of technologies uses this `#8993A4` color and not `black`. Kindly change that so it fits with that of the Figma design. 

### Your design 
image

### Figma Design
image

File: index.html [hunk 5]
```diff
@@ -1,47 +1,60 @@
...
     <a href="index.html" class="logo">
       <h1>Byiringiro Moise</h1>
     </a>
     <span class="material-symbols-outlined"> menu </span>
+    <div class="nav-lists">
+      <a href="">Profile</a>
+      <a href="">About</a>
+      <a href="">Contact</a>
+      <a href=""><span class="material-symbols-outlined"> mail </span></a>
+    </div>
```
Comment (bobb-Rob) [inline index.html:26]: - [ ] [REQUIRED] Your navbar navigation items are missing. I noticed it was present when you submitted it for the first review. But now, it is gone. Kindly look into this and fix it. 
image

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️

Hi @Moise-code , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gopxfs** [comment]
# 🟢 STATUS: APPROVED 🟢

## Hello @Moise-code 👋,

Your project is complete! You followed all the requirements, feel free to merge it! 😄
Congratulations! 🎉 



## Highlights
✔️ Linters are passing successfully.
✔️ Implemented all the changes requested by the previous reviewer. Well done!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**